content/en/blog/2022/otel-demo-app-nomad/index.md

```diff
@@ -0,0 +1,383 @@
+  [Nomad intro post](https://adri-v.medium.com/just-in-time-nomad-80f57cd403ca).
+  [This blog post](https://danielabaron.me/blog/nomad-tips-and-tricks/) by
+  [Daniela Baron](https://danielabaron.me) is also great.
+- **Observability (o11y) and [OpenTelemetry](https://opentelemetry.io) (OTel)**.
+  If not, mozy on over to my
+  [Observability & OTel post](https://storiesfromtheherd.com/unpacking-observability-the-observability-stack-93d4733e2a72).
+
+### Pre-Requisites
+
+In order to run the example in this tutorial, you’ll need the following:
+
+- [Docker](https://docker.com) (version 20.10.21 at the time of this writing)
+- [Vagrant](https://vagrantup.com) (version 2.3.1 at the time of this writing)
+
+### Tutorial Repos
+
+Below are the repos that we’ll be using for today’s tutorial:
+
+- My modified [HashiQube Repo](https://github.com/avillela/hashiqube) (fork of
+  [servian/hashiqube](https://github.com/servian/hashiqube)). If you’re curious,
+  you can see what modifications I’ve made
+  [here](https://github.com/avillela/hashiqube).
+- My [Nomad Conversions](https://github.com/avillela/nomad-conversions) repo
+
+### HashiQube Setup
+
+Before you start, just a friendly reminder that HashiQube by default runs
+[Nomad](https://nomadproject.io), [Vault](http://vaultproject.io), and
+[Consul](https://consul.io) on Docker. In addition, we’ll be deploying 21 job
+specs to Nomad. This means that we’ll need a decent amount of CPU and RAM, so
+please make sure that you have enough resources allocated in your Docker
+desktop. For reference, I’m running an M1 Macbook Pro with 8 cores and 32 GB
+RAM. My Docker Desktop Resource settings are as follows:
+
+- **CPUs:** 3
+- **Memory:** 9.5GB
+- **Swap:** 3GB
+
+Here’s a screenshot of my Docker Preferences Resources settings, if you need a
+visual:
+
+![Screen capture of Docker Desktop resources preferences](docker-desktop-resources-config.png "Screen capture of Docker Desktop resources preferences")
+
+For more, check out the Docker docs on how to change your resources settings for
+[Mac](https://docs.docker.com/desktop/settings/mac/),
+[Windows](https://docs.docker.com/desktop/settings/windows/), and
+[Linux](https://docs.docker.com/desktop/settings/linux/).
+
+#### 1- Update /etc/hosts
+
+We use the [Traefik](https://traefik.io) load-balancer to expose our services,
+which we access as subdomains of localhost. In order ensure that we can access
+our Traefik-exposed services (and also the Traefik dashboard itself, you’ll need
+to add the following entries to `/etc/hosts` on your host machine:
+
+```
+127.0.0.1   traefik.localhost
+127.0.0.1   otel-demo.localhost
+```
+
+#### 2- Provision a Local Hashi Environment with HashiQube
+
+Start HashiQube by following the detailed instructions
+[here](https://github.com/avillela/hashiqube#quickstart).
+
+> **NOTE:** Be sure to check out the
+> [Gotchas](https://github.com/avillela/hashiqube#gotchas) section, if you get
+> stuck.
```

Comment:
I think this way of creating a note only works with github markdown, can you check some other places on opentelemetry.io that have a "Note" and use that accordingly

Reply:
I found [another blog post](https://opentelemetry.io/blog/2022/exponential-histograms/) that uses this for notes, and it appears to render properly on the site.